### PR TITLE
Pre launch styling tweaks

### PR DESCRIPTION
**Pull request by sonicyeti**
Various styling updates to ensure parity with latest design specifications.

- Card title padding fix
- Card tag removal if “unknown” type
- Header padding, logo size, border radius

Screenshots:
Screen Shot 2020-04-28 at 10 32 50 AM
Screen Shot 2020-04-28 at 10 32 56 AM

### CODE REVIEW

**Comment by nijikokun** [comment]
> Card tag removal if “unknown” type

**Review by jitachi — DISMISSED** [review]

File: packages/insomnia-app/app/ui/components/wrapper-design.js [hunk 1]
```diff
@@ -203,8 +203,8 @@ class WrapperDesign extends React.PureComponent<Props, State> {
             className="app-header"
             gridLeft={
               <React.Fragment>
-                <img src={designerLogo} alt="Insomnia" width="24" height="24" />
-                <Breadcrumb className="breadcrumb" crumbs={['Documents', activeApiSpec.fileName]} onClick={this._handleBreadcrumb} />
+                <img src={designerLogo} alt="Insomnia" width="32" height="32" />
+                <Breadcrumb className="breadcrumb" crumbs={['Documents', activeWorkspace.name]} onClick={this._handleBreadcrumb} />
```
Comment (develohpanda) [inline packages/insomnia-app/app/ui/components/wrapper-design.js]:
```suggestion
                <Breadcrumb className="breadcrumb" crumbs={['Documents', activeApiSpec.fileName]} onClick={this._handleBreadcrumb} />
```

File: packages/insomnia-app/app/ui/components/wrapper-home.js [hunk 2]
```diff
@@ -349,7 +349,7 @@ class WrapperHome extends React.PureComponent<Props, State> {
     let defaultActivity = ACTIVITY_DEBUG;
 
     if (spec || w.scope === 'spec') {
-      label = 'Unknown';
+      let label: string = '';
```
Comment (develohpanda) [inline packages/insomnia-app/app/ui/components/wrapper-home.js]:
```suggestion
      label = 'Unknown';
```

**Review by develohpanda — COMMENTED** [review]
LGTM!

**Pull request closed without merging**